ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -320,6 +311,17 @@ post_provision_config_nodes() {
         dnf -y config-manager --add-repo="${repo_url}"
         disable_gpg_check "$repo_url"
     done
+
+    if ! "${cmd[@]}" --exclude golang-*.daos.*; then
+	    dump_repos
+	    return 1
+    fi
+
+    if lspci | grep "ConnectX-6" && ! grep MOFED_VERSION /etc/do-release; then
+	    # Remove OPA and install MOFED
+	    install_mofed
+    fi
+
```

Comment:
ok. Shall I restart the test with the change? Or it does not matter since this is only for testing go PR? Thank you!

Reply:
Since you need to make the above change, let's revert the move of https://github.com/daos-stack/daos/blame/990a3ae6d260719ff1be84c21167f7d205c9734c/ci/provisioning/post_provision_config_common_functions.sh#L320-L324 down here.